
<!DOCTYPE HTML>

<html lang="en">

<head>
  <meta charset="utf-8">
  <title>HandleExceptions (Foundation&#92;Bootstrap) - Laravel 5.2 - W3cubDocs</title>
  
  <meta name="description" content=" class HandleExceptions (View source) ">
  <meta name="keywords" content="handleexceptions, foundation, bootstrap, -, laravel, laravel~5.2">
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="mobile-web-app-capable" content="yes">
  
  <link rel="canonical" href="http://docs.w3cub.com/laravel~5.2/api/5.2/illuminate/foundation/bootstrap/handleexceptions/">
  <link href="/favicon.png" rel="icon">
  <link type="text/css" rel="stylesheet" href="/assets/application-50364fff564ce3b6327021805f3f00e2957b441cf27f576a7dd4ff63bbc47047.css">
  <script type="text/javascript" src="/assets/application-db64bfd54ceb42be11af7995804cf4902548419ceb79d509b0b7d62c22d98e6f.js"></script>
  <script src="/json/laravel~5.2.js"></script>
  
  <script type="text/javascript">
    var _gaq = _gaq || [];
    _gaq.push(['_setAccount', 'UA-71174418-1']);
    _gaq.push(['_trackPageview']);

    (function() {
      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
    })();
  </script>


</head>

<body>
	<div class="_app">
	<header class="_header">
  
  <form class="_search">
    <input type="search" class="_search-input" placeholder="Search&hellip;" autocomplete="off" autocapitalize="off" autocorrect="off" spellcheck="false" maxlength="20">
    <a class="_search-clear"></a>
    <div class="_search-tag"></div>
  </form>
  
  <a class="_home-link" href="/" ></a>
  <a class="_menu-link"></a>
  <h1 class="_logo">
    <a href="/" class="_nav-link" title="API Documentation Browser">W3cubDocs</a>
  </h1>
  
  <span class="_logo-sub-nav">/</span><span class="_logo-sub-nav"><a href="/laravel~5.2/" class="_nav-link" title="" style="margin-left:0;">Laravel 5.2</a></span>
  
  <nav class="_nav">
    <a href="/app/" class="_nav-link ">App</a>
    <a href="/about/" class="_nav-link ">About</a>
  </nav>
</header>
	<section class="_sidebar">
		<div class="_list">
			
		</div>
	</section>
	<section class="_container ">
		<div class="_content">
			<div class="_page _laravel">
				
<h1>HandleExceptions</h1> <p> class <strong>HandleExceptions</strong> (<a href="https://github.com/laravel/framework/blob/5.2/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php" target="_blank">View source</a>) </p> <h2>Methods</h2> <table class="container-fluid underlined"> <tr class="row"> <td class="col-md-2 type"> void </td> <td class="col-md-8 type"> <a href="#method_bootstrap">bootstrap</a>( <a href="../../../contracts/foundation/application/" title="Illuminate\Contracts\Foundation\Application">Application</a> $app) <p>Bootstrap the given application.</p> </td> <td class="col-md-2"></td> </tr> <tr class="row"> <td class="col-md-2 type"> void </td> <td class="col-md-8 type"> <a href="#method_handleError">handleError</a>( int $level, string $message, string $file = '', int $line, array $context = array()) <p>Convert a PHP error to an ErrorException.</p> </td> <td class="col-md-2"></td> </tr> <tr class="row"> <td class="col-md-2 type"> void </td> <td class="col-md-8 type"> <a href="#method_handleException">handleException</a>( <a target="_blank" href="http://php.net/Throwable">Throwable</a> $e) <p>Handle an uncaught exception from the application.</p> </td> <td class="col-md-2"></td> </tr> <tr class="row"> <td class="col-md-2 type"> void </td> <td class="col-md-8 type"> <a href="#method_handleShutdown">handleShutdown</a>() <p>Handle the PHP shutdown event.</p> </td> <td class="col-md-2"></td> </tr> </table> <h2>Details</h2>  <div class="method-item"> <h3 id="method_bootstrap">                      
    void
    bootstrap(
    <a href="../../../contracts/foundation/application/" title="Illuminate\Contracts\Foundation\Application">Application</a> $app) </h3> <div class="details"> <div class="method-description"> <p>Bootstrap the given application.</p> </div> <div class="tags"> <h4>Parameters</h4> <table class="table table-condensed"> <tr> <td> <a href="../../../contracts/foundation/application/" title="Illuminate\Contracts\Foundation\Application">Application</a>
</td> <td>$app</td> <td></td> </tr> </table> <h4>Return Value</h4> <table class="table table-condensed"> <tr> <td> void</td> <td></td> </tr> </table> </div> </div> </div> <div class="method-item"> <h3 id="method_handleError">                      
    void
    handleError(
    int $level, 
    string $message, 
    string $file = '', 
    int $line, 
    array $context = array()) </h3> <div class="details"> <div class="method-description"> <p>Convert a PHP error to an ErrorException.</p> </div> <div class="tags"> <h4>Parameters</h4> <table class="table table-condensed"> <tr> <td> int</td> <td>$level</td> <td></td> </tr> <tr> <td> string</td> <td>$message</td> <td></td> </tr> <tr> <td> string</td> <td>$file</td> <td></td> </tr> <tr> <td> int</td> <td>$line</td> <td></td> </tr> <tr> <td> array</td> <td>$context</td> <td></td> </tr> </table> <h4>Return Value</h4> <table class="table table-condensed"> <tr> <td> void</td> <td></td> </tr> </table> <h4>Exceptions</h4> <table class="table table-condensed"> <tr> <td><a target="_blank" href="http://php.net/ErrorException">ErrorException</a></td> <td></td> </tr> </table> </div> </div> </div> <div class="method-item"> <h3 id="method_handleException">                      
    void
    handleException(
    <a target="_blank" href="http://php.net/Throwable">Throwable</a> $e) </h3> <div class="details"> <div class="method-description"> <p>Handle an uncaught exception from the application.</p> <p>Note: Most exceptions can be handled via the try / catch block in the HTTP and Console kernels. But, fatal error exceptions must be handled differently since they are not normal exceptions.</p> </div> <div class="tags"> <h4>Parameters</h4> <table class="table table-condensed"> <tr> <td> <a target="_blank" href="http://php.net/Throwable">Throwable</a>
</td> <td>$e</td> <td></td> </tr> </table> <h4>Return Value</h4> <table class="table table-condensed"> <tr> <td> void</td> <td></td> </tr> </table> </div> </div> </div> <div class="method-item"> <h3 id="method_handleShutdown">                      
    void
    handleShutdown() </h3> <div class="details"> <div class="method-description"> <p>Handle the PHP shutdown event.</p> </div> <div class="tags"> <h4>Return Value</h4> <table class="table table-condensed"> <tr> <td> void</td> <td></td> </tr> </table> </div> </div> </div>
<div class="_attribution">
  <p class="_attribution-p">
    Â© Taylor Otwell<br>Licensed under the MIT License.<br>Laravel is a trademark of Taylor Otwell.<br>
    <a href="https://laravel.com/api/5.2/Illuminate/Foundation/Bootstrap/HandleExceptions.html" class="_attribution-link" target="_blank">https://laravel.com/api/5.2/Illuminate/Foundation/Bootstrap/HandleExceptions.html</a>
  </p>
</div>

			</div>
		</div>
	</section>

	</div>
</body>
</html>
